lib/Transforms/CMakeLists.txt

```
@@ -0,0 +1,13 @@
+add_subdirectory(ApplyFolders)
+add_subdirectory(ConvertIfToSelect)
+add_subdirectory(ElementwiseToAffine)
+add_subdirectory(ForwardStoreToLoad)
+add_subdirectory(FullLoopUnroll)
+add_subdirectory(Secretize)
+add_subdirectory(StraightLineVectorizer)
+add_subdirectory(UnusedMemref)
+
+if(ENABLE_YOSYS)
+    message(FATAL_ERROR "Yosys transforms not supported in CMake build yet; use Bazel instead.")
```

Can you create an issue for this and link to it here + anywhere else changes would have to be made? 
(there's a CI thing that keeps track of mentions for each issue)
```suggestion
    # TODO (#999): Support Yosys in cmake build.
    message(FATAL_ERROR "Yosys transforms not supported in CMake build yet; use Bazel instead.")
```